modules/multi-architecture-modify-machine-set-gcp.adoc

```diff
@@ -0,0 +1,150 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-modify-machine-set-gcp_{context}"]
+
+= Adding an ARM64 compute machine set to your GCP cluster 
+
+To configure a cluster with multi-architecture compute machines, you must create a GCP ARM64 compute machine set. This adds ARM64 compute nodes to your cluster so that your cluster has multi-architecture compute machines.
```

Comment:
```suggestion
To configure a cluster with multi-architecture compute machines, you must create a GCP ARM64 compute machine set. This adds ARM64 compute nodes to your cluster.
```
Suggestion - you could probably leave this duplication of the first sentence off.

Reply:
Great suggestion, yeah its a bit redundant when included